src/Aimeos/Shop/Controller/JqadmController.php

```diff
@@ -257,6 +257,17 @@ protected function createAdmin() : \Aimeos\Admin\JQAdm\Iface
 		$context = app( 'aimeos.context' )->get( false, 'backend' );
 		$context->setI18n( app( 'aimeos.i18n' )->get( array( $lang, 'en' ) ) );
 		$context->setLocale( app( 'aimeos.locale' )->getBackend( $context, $site ) );
+		
+		$siteManager = \Aimeos\MShop::create( $context, 'locale/site');
+
+		$filter = $siteManager->filter();
+		$filter->add([ 'locale.site.code' => $site ]);
+    
+		$siteItem = $siteManager->search( $filter->slice( 0 ) );
```

Comment:
You can use `$siteManager->find( $site )` instead

Reply:
Excellent, that is much better. I have switched to that method.